### ISSUE TITLE
Can it download hls streaming link

### ISSUE THREAD

**Issue by HiddenPirates** [amitshekhariitbhu/PRDownloader#203]
Can I download m3u8 link with this library?

**Comment by suleman81** [comment]
No you can't i tried

**Comment by neogen-oss** [comment]
No, only single normal file. If you give HAS link to it, it will download the hls file, witch only contains keyframes, locations of segments, not a video or audio file.
